server/cmd/dcrdex/main.go

```diff
@@ -6,19 +6,19 @@ package main
 import (
 	"context"
 	"fmt"
+	"io/ioutil"
 	"net/http"
 	_ "net/http/pprof"
 	"os"
+	"path/filepath"
+	"plugin"
 	"runtime"
 	"runtime/pprof"
 	"strings"
 	"sync"
 
 	"decred.org/dcrdex/dex/encode"
 	"decred.org/dcrdex/server/admin"
-	_ "decred.org/dcrdex/server/asset/btc" // register btc asset
-	_ "decred.org/dcrdex/server/asset/dcr" // register dcr asset
-	_ "decred.org/dcrdex/server/asset/ltc" // register ltc asset
```

Comment:
Eliminating three lines and the need to edit this file to add an additional asset seems to be the one and only thing we get from plugins given the requirement of identical build environments, we'll have to see how we like this when we start using it.